apps/demos/Demos/DropDownButton/Overview/Angular/app/app.component.css

```diff
@@ -16,3 +16,54 @@
   cursor: pointer;
   font-size: 18px;
 }
+
+.dx-button-wrapper {
+  padding-inline-start: 12px;
+  padding-inline-end: 4px;
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+}
+
+.button-img-wrapper {
+  position: relative;
+  height: 32px;
+}
+
+.dx-button-img {
+  width: 32px;
+  height: 32px;
+  position: relative;
+  border-width: 1px;
+  border-style: solid;
+  border-color: var(--dx-color-border);
+  border-radius: 50%;
+}
+
+.button-img-indicator {
+  position: absolute;
+  background-color: var(--dx-color-danger);
+  top: -1px;
+  right: -1px;
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+  border-width: 2px;
+  border-style: solid;
+  border-color: var(--dx-color-main-bg);
+  z-index: 1;
+}
+
+.dx-button-title {
+  line-height: 20px;
+}
+
+.dx-button-row {
+  font-size: 12px;
+  line-height: 14px;
+  opacity: 0.6;
+}
+
+::ng-deep #two-sections .dx-button {
```

Comment:
#two-sections is id of dropDownButton component. I think the id is not accurately describe the component's purpose, functionality, or content, you might consider choosing a more descriptive ID. 
@vladaskorohodova Could you please check class, Ids names that are used?

Reply:
I would use "container" instead of "wrapper".
button-img-wrapper -> button-img-container
button-wrapper -> text-container

Also, why do some classes have a "dx-" prefix, and some do not? Not consistent.

When it comes to "two-sections", I would rename it to "custom-template".